src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -3,8 +3,31 @@ title: React
 sidebar_order: 30
 ---
 <!-- WIZARD -->
-To use Sentry with your React application, you will need to use `@sentry/browser` (Sentry’s browser JavaScript SDK).  
-On its own, `@sentry/browser` will report any uncaught exceptions triggered from your application.
+To use Sentry with your React application, you will need to use Sentry’s browser JavaScript SDK: `@sentry/browser`.
+
+```bash
+# Using yarn
+$ yarn add @sentry/browser
+
+# Using npm
+$ npm install @sentry/browser
```

Comment:
Adds how to actually install the sdk for react.

Might try and split this part into a more general section in the wizard?